### ISSUE TITLE
ModuleNotFoundError: No module named 'mmdet.datasets.pipelines.formatting'

### ISSUE THREAD

**Issue by Hidayathamir** [open-mmlab/mmocr#965]
**Checklist**

1. Yes. I have searched related issues but cannot get the expected help.
2. Yes. The bug has not been fixed in the latest version.

**Describe the bug**
I have not done anything yet, just clone the repo, build docker image, run the image, then do text detection that [documentation](https://mmocr.readthedocs.io/en/latest/demo.html#example-1-text-detection) tell me, but I got this error `ModuleNotFoundError: No module named 'mmdet.datasets.pipelines.formatting'`

**Reproduction**

1. What command or script did you run?
```
git clone https://github.com/open-mmlab/mmocr.git

cd mmocr/

docker build -t mmocr docker/

docker run -itd --gpus all --name mmocr --shm-size=8g -v /datadrive/mmocr:/mmocr mmocr
```
I am attach to running container with vscode extention [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers). Inside the container, I run this command.
```none
python mmocr/utils/ocr.py demo/demo_text_det.jpg --output demo/det_out.jpg --det TextSnake --recog None --export demo/
```
Then I got this error `ModuleNotFoundError: No module named 'mmdet.datasets.pipelines.formatting'`.

2. Did you make any modifications on the code or config? No.
3. What dataset did you use? `demo/demo_text_det.jpg`.

**Environment**

1. Please run `python mmocr/utils/collect_env.py` to collect necessary environment information and paste it here.
```
root@43f06e5422f8:/mmocr# python mmocr/utils/collect_env.py
sys.platform: linux
Python: 3.7.7 (default, May  7 2020, 21:25:33) [GCC 7.3.0]
CUDA available: True
GPU 0: Tesla T4
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 10.1, V10.1.24
GCC: gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
PyTorch: 1.6.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.1 Product Build 20200208 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.5.0 (Git Hash e2ac1fac44c5078ca927cb9b90e1b3066a0b2ed0)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.7.0
OpenCV: 4.5.4
MMCV: 1.5.0
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 10.1
MMOCR: 0.5.0+888f700
```

**Error traceback**
If applicable, paste the error traceback here.

```none
root@43f06e5422f8:/mmocr# python mmocr/utils/ocr.py demo/demo_text_det.jpg --output demo/det_out.jpg --det TextSnake --recog None --export demo/
Traceback (most recent call last):
  File "mmocr/utils/ocr.py", line 25, in <module>
    from mmocr.datasets.kie_dataset import KIEDataset
  File "/mmocr/mmocr/datasets/__init__.py", line 7, in <module>
    from .kie_dataset import KIEDataset
  File "/mmocr/mmocr/datasets/kie_dataset.py", line 12, in <module>
    from mmocr.datasets.pipelines import sort_vertex8
  File "/mmocr/mmocr/datasets/pipelines/__init__.py", line 3, in <module>
    from .custom_format_bundle import CustomFormatBundle
  File "/mmocr/mmocr/datasets/pipelines/custom_format_bundle.py", line 5, in <module>
    from mmdet.datasets.pipelines.formatting import DefaultFormatBundle
ModuleNotFoundError: No module named 'mmdet.datasets.pipelines.formatting'
```

**Bug fix**
I found this [commit](https://github.com/open-mmlab/mmocr/commit/66880d2b290c6d0c0fc8d102047d45900eecfc1a) that change from `from mmdet.datasets.pipelines.formating import DefaultFormatBundle` to `from mmdet.datasets.pipelines.formatting import DefaultFormatBundle`. When I change `formatting` back to the original `formating`, all works fine.

**Comment by Mountchicken** [comment]
Hi @Hidayathamir 
Maybe there is something wrong with docker. I was able to run your command properly after installing MMOCR using the following steps.
```bash
git clone https://github.com/open-mmlab/mmocr.git
cd mmocr

pip install -r requirements.txt
pip install -v -e .  # or "python setup.py develop"
export PYTHONPATH=$(pwd):$PYTHONPATH
```

**Comment by gaotongxiao** [comment]
Fixed in #966
